qemu/tests/qemu_guest_agent.py

```diff
@@ -1043,7 +1047,42 @@ def _change_bl():
         self.gagent.guest_file_seek(ret_handle, -5, 2)
         _read_check(ret_handle, "orl", 3)
 
-        error_context.context("Close the opened file.", logging.info)
+        self.gagent.guest_file_close(ret_handle)
+
+        # create a new file by agent.
+        error_context.context("Create new file with mode 'w' and do file write"
+                              " test", logging.info)
+        ranstr = utils_misc.generate_random_string(5)
+        tmp_file = "%s%s" % (params["file_path"], ranstr)
+        ret_handle = int(self.gagent.guest_file_open(tmp_file, mode="w+"))
+
+        error_context.context("Write two counts bytes to guest file.",
```

Comment:
`two counts of byte`?

Reply:
It should be two bytes. Will update.thanks